setup.sh

```diff
@@ -42,70 +43,94 @@ fi
 }
 
 create_bucket() {
-        if [ "${USE_AWS_VAULT}" = 'true' ] ; then  
-                aws-vault exec ${PROFILE_NAME} -- aws s3 mb "s3://${TERRAFORM_BUCKET}"
-                aws-vault exec ${PROFILE_NAME} -- aws s3api put-bucket-versioning  \
-                --bucket ${TERRAFORM_BUCKET} \
-                --versioning-configuration Status=Enabled
+# Creates versioned AWS bucket to store remote terraform state
+        aws-vault exec ${PROFILE_NAME} -- aws s3 mb "s3://${TERRAFORM_BUCKET}"
+        aws-vault exec ${PROFILE_NAME} -- aws s3api put-bucket-versioning  \
+        --bucket ${TERRAFORM_BUCKET} \
+        --versioning-configuration Status=Enabled
+}
+
+does_stack_config_exist() {
+        if [ -e "${ROOTPROJ}/stacks/${ENV}.backend" ] ; then
+                return "1"
```

Comment:
it's confusing to use `1` to mean "true" in bash.. normally `0` is true and any nonzero value is false.

Reply:
Thats true, will switch them around